src/test/java/org/generationcp/middleware/dao/ProjectDAOTest.java

```diff
@@ -135,36 +135,6 @@ public void setup() {
 
 	}
 
-	@Test
```

Comment:
why the tests that are about project are removed?
@cuenyad

Reply:
Reverted change and fixied test.